src/app/news/news-list-item.component.ts

```diff
@@ -47,6 +49,7 @@ export class NewsListItemComponent implements OnChanges, AfterViewChecked {
   ngOnChanges() {
     this.targetLocalPlanet = this.shareTarget === this.stateService.configuration.planetType;
     this.showShare = this.shouldShowShare();
+    this.isActiveMember = !this.shouldShowIndication();
```

Comment:
No need to make this a `!` of the function, especially considering it is then reversed again in the template (see my other comment, you should change that as well).

Generally if you are reversing a boolean you should take a minute to think of ways you could just flip your condition.  If not that's fine, but in this case since the function and the value are only used in one place you should be able to.